.github/workflows/test-race.yml

```diff
@@ -0,0 +1,101 @@
+name: Tests Race
+#  Tests Race workflow runs unit tests with the race detector
+#  This workflow is run on pushes to master & every Pull Requests where a .go, .mod, .sum have been changed
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  install-tparse:
+    runs-on: ubuntu-latest
```

Comment:
do we really need to use `ubuntu`? Alpine is much smaller, so it should load faster.

Reply:
https://github.com/actions/virtual-environments this isn't a docker image.